ONSdigital/eq-questionnaire-runner
# Contain visible labels within accessible names

### What is the context of this PR?
The `Start section`, `Continue with section` and `View answers` links in hubs were not fully accessible to voice activation users because visible labels of these links are not contained within their accessible names, for example ‘Start Household bills section’ doesn't exactly contain it's visible label of `Start section` which could be confusing.

In order to update this, the `aria_labels` within `SECTION_CONTENT_STATES` had to be updated to change the order of which the label/section name is placed. Previously, it was ‘Start {section_name} section’ and the new version is now `Start section: {section_name}`, the same goes for the other links.

Tests were updated accordingly, a missing unit test for an `IN_PROGRESS` section was added and functional tests included a check for the html to ensure the visually hidden text is as expected.

### How to review
- Ensure the changes made are suitable
- Check the changes to the functional tests and if more changes are required
- Check all tests (functional & unit) pass
- Open any schema with a hub and see if the voiceover works as intended. A good schema to test all variants of the updated visually hidden text is `test_grand_calculated_summary_cross_section_dependencies`. Answer one question of the `Household bills` section, then return to the hub, then complete the `Grand Calculated Summary` section resulting in a hub with all 3 link variants. Use the voiceover (Shortcut command: cmd+fn+f5) and make sure the text is read out correctly, i.e. `Start section: Household bills`. Feel free to test other schemas with a hub.

### Checklist
